website/content/docs/k8s/annotations-and-labels.mdx

```diff
@@ -74,83 +74,96 @@ The following Kubernetes resource annotations could be used on a pod to control
 - `consul.hashicorp.com/connect-service-upstreams` - The list of upstream
   services that this pod needs to connect to via the service mesh along with a static
   local port to listen for those connections. When transparent proxy is enabled,
-  this annotation is optional. There are a few formats this annotation can take:
+  this annotation is optional. This annotation can be either _unlabeled_ or _labeled_. Unlabeled annotations use component names, and the component types are inferred through the annotation's syntax. Labeled annotations explicitly associate component names and types to form Consul DNS addresses. To configure upstreams for [cluster peering](/consul/docs/k8s/connect/cluster-peering/tech-specs), use the labeled format.
 
-    - Unlabeled:
+    - **Unlabeled**:
        Use the unlabeled annotation format to specify a service name, Consul Enterprise namespaces and partitions, and
```

Comment:
This pairs with the suggested changes to the labeled section.

```suggestion
       The unlabeled annotation format allows you to reference any service not in a cluster peer as an upstream. You can specify a Consul Enterprise namespace. You can also specify an admin partition in the same datacenter or a WAN federated datacenter. Unlike the labeled annotation, you can also reference a prepared query as an upstream.
```

I hope to publish some changes to the docs that include an explanation on why prepared queries are not recommended with service mesh... but because that content isn't available to link to yet, I don't want to try to explain why prepared queries aren't recommended as upstreams here. I'll address that in my (future) PR.